docs/specs/build/config/compat-include.yml

```diff
@@ -0,0 +1,25 @@
+---
```

Comment:
consider merge content include/exclude test inside a single file

Reply:
For easy debug... will merge after test framework improvement

---
In reply to: [184842399](https://github.com/dotnet/docfx/pull/2702#discussion_r184842399) [](ancestors = 184842399)